modules/administration/pages/custom-ssl.adoc

```diff
@@ -1,97 +1,289 @@
+
+. On the client, create a new custom channel using these details:
 +
-Proceed with the default setup.
-Upon reaching the Certificate Setup window during {yast}
-installation, fill in random values, as these will be overridden with the values specified in <<bp.cert.custom.setup.proc.export>>.
+* Name: SSL-CA-Channel
+* Label: ssl-ca-channel
+* Parent Channel: <choose the parent channel of a clients>
+* Summary: SSL-CA-Channel
+
 +
-.Shell Requirements
-NOTE: Make sure that you execute [command]``yast2
-      susemanagersetup`` from within the same shell the environment variables were exported from.
+For more on creating custom channels, see xref:administration:channel-management.adoc[].
+. Upload the CA certificate RPM to the channel:
 +
+----
+rhnpush -c ssl-ca-channel --nosig \
+--ca-chain=/srv/www/htdocs/pub/RHN-ORG-TRUSTED-SSL-CERT \
+/root/combined-ssl-build/rhn-org-trusted-ssl-cert-1.0-2.noarch.rpm
+----
+. Subscribe all clients to the new ``SSL-CA-Channel`` channel.
+. Install the CA certificate RPM on all clients by updating the channel.
+
+
 
+.Procedure: Deploying Combined Certificates on Salt Clients
 
+// Does the above procedure also work for Salt? LKB 2019-09-10
 
-[[bp.cert.custom.proxy]]
-== Using a Custom Certificate with {susemgrproxy}
+. On the client, in the {productname} {webui}, navigate to menu:Salt[Remote Commands].
+. In the first field, enter [guimenu]``salt-call state.apply certs``.
+In the second field, after the ``@``, enter [guimenu]``*``.
+Click btn:[Find Targets].
+. Check that the appropriate clients are displayed, and click btn:[Run Command] to deploy the certificates on the clients.
 
+image::deploy_cert_salt.png[scaledwidth=80%]
 
-After completing the installation with yast found in <<advanced.topics.proxy.quickstart>> continue with a modified <<at.manager.proxy.run.confproxy>> procedure:
+When you have every client trusting both the old and new certificates, you can go ahead and replace the server certificate on the {productname} Server and Proxies.
 
-[[bp.cert.custom.proxy.proc]]
 
-. Execute [command]``configure-proxy.sh``.
-. When prompted with:
+
+.Procedure: Replace Server Certificate on the Server
+
+. On the {productname} Server, at the command prompt, install the RPM from the [path]``ssl-build`` directory:
++
+----
+rpm -Uhv ssl-build/susemanager/rhn-org-httpd-ssl-key-pair-susemanager-1.0-2.noarch.rpm
+----
+. Restart services to pick the changes:
 +
+----
+spacewalk-service restart
+----
+
+
+
+.Procedure: Replace Server Certificate on the Proxy
 
+. On the {productname} Proxy, at the command prompt, install the RPM from the [path]``ssl-build`` directory:
++
 ----
-Do you want to import existing certificates?
+rpm -Uhv ssl-build/susemanager-proxy/rhn-org-httpd-ssl-key-pair-susemanager-proxy-1.0-2.noarch.rpm
 ----
+. Restart services to pick up the changes:
 +
-Answer with kbd:[y]
-.
-. Continue by following the script prompts.
+----
+spacewalk-service restart
```

Comment:
```suggestion
rhn-proxy restart
```

For proxies we have a different command. And we do not have a symlink which name it mgr-proxy :-(